lib/matplotlib/tests/test_axes.py

```diff
@@ -4072,6 +4072,19 @@ def test_axline(fig_test, fig_ref):
     ax.axvline(-0.5, color='C5')
 
 
+@check_figures_equal()
+def test_axline_transform(fig_test, fig_ref):
```

Comment:
I think it would be useful to add a test where you call the method, and then subsequently resize the figure (simulating a manual resize) and show this still yields the expected line.  Also that its robust to panning and zooming.

Reply:
Sure. I added some an additional test in 45e5bc7 which uses `set_xlim`/`set_ylim` and `set_size_inches` to simulate resize and pan/zoom, is that what you meant?